Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/VehicleRegistrationCertificate.hs

```diff
@@ -282,6 +298,13 @@ linkRCStatus (driverId, merchantId) req@RCStatusReq {..} = do
       deactivateRC rc driverId
   return Success
 
+updateRC :: (Id Person.Person, Id DM.Merchant) -> UpdateRCReq -> Flow APISuccess
+updateRC (driverId, merchantId) UpdateRCReq {..} = do
+  (rc, now) <- processRC driverId merchantId rcNo
+  validateRCActivation driverId merchantId rc
+  VQuery.updateVehicleColorAndVariant (fromMaybe "Unkown" rc.vehicleColor) (fromMaybe AUTO_RICKSHAW rc.vehicleVariant) driverId now
```

Comment:
Why defaults are "Unkown` and AUTO_RICKSHAW, it should be the same vehicleVariant which current vehicle is having.